RELEASENOTES.md

```diff
@@ -14,6 +14,41 @@ If you are using Red Hat Enterprise Linux 7 or CentOS 7, you can [update using `
 
 # Releases
 
+## Version 1.2.0
+
+The `rippled` 1.2.0 release introduces the MultisignReserve Amendment, which
+reduces the reserve requirement associated with signer lists. This release also
+includes incremental improvements to the code that handles offers. Furthermore,
+`rippled` now also has the ability to automatically detect transaction
+censorship attempts and issue warnings of increasing severity for transactions
+that should have been included in a closed ledger after several rounds of
+consensus.
+
+**New and Updated Features**
+
+- Reduce the account reserve for a Multisign SignerList (6572fc8)
+- Improve transaction error condition handling (4104778)
+- Allow servers to automatically detect transaction censorship attempts (945493d)
+- Load validator list from file (c1a0244)
+- Add RPC command shard crawl (17e0d09)
+- Add RPC Call unit tests (eeb9d92)
+- Grow the open ledger expected transactions quickly (7295cf9)
+- Avoid dispatching multiple fetch pack threads (4dcb3c9)
+- Remove unused function in AutoSocket.h (8dd8433)
```

Comment:
I don't consider this a "new and updated feature".

Maybe we should have a third category for "under-the-hood improvements" or a similar name.